projects/storefrontlib/src/cms-components/myaccount/update-email/update-email.component.html

```diff
@@ -1,20 +1,85 @@
-<ng-container>
-  <div *ngIf="isLoading$ | async; else loaded">
-    <div class="cx-spinner">
-      <cx-spinner> </cx-spinner>
-    </div>
-  </div>
-
-  <ng-template #loaded>
-    <div class="container">
-      <div class="row d-flex justify-content-center">
-        <cx-update-email-form
-          class="col-md-6"
-          (saveEmail)="onSubmit($event)"
-          (cancelEmail)="onCancel()"
-        >
-        </cx-update-email-form>
+<div class="container" [ngClass]="{'is-loading': isLoading$ | async}">
+  <div class="row d-flex justify-content-center">
```

Comment:
I prefer to drop the bootstrap classes here:
- polluting the html
- forces multiple wrapping elements (div>div>div)
- "blocks" customers from changing the UI easily (i.e. say you don't like to center the form?).